copier.yml

```
@@ -1,3 +1,9 @@
+# Questions specific to this template
+
+
+
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (5 > 2)

(empty-lines)